### DISCUSSION TITLE
Adaptive card support for "Send an email" Outlook Action

### DISCUSSION THREAD

**Discussion by bckrma** [Azure/logicapps#679] [Feature Ideas/Requests]
Hello,

it would be great if the "Send an Email" Outlook connector would support adaptive cards like the one in powerautomate does.
Currently the connector will send the e-mail with the adaptive card payload (as long as this was created in a compose action), but outlook will not render the email.
The same payload works in PowerAutomate.

This would be great to send the same notifications to Teams and E-Mail Recipients.

Thanks a lot!

**Comment by willheineman** [comment]
Did you find any workaround for this? Currently facing the same issue and I do not want to use the Send Approval action in logic apps.
